### DISCUSSION TITLE
Shortcuts for GitHub

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [pfmc-assessments/sa4ss#93] [Show and tell]
Well, as always, I am late 🙀. This post was meant for last week, which is why I am always in the need of shortcuts while on my long road of to dos. Often, I find shortcuts hard to remember and a waste of time to learn. For some reason though, I :heart: the following three shortcuts on GitHub
* `g i` - will take you directly to the issues page for a repository
* `g g` - will take you to the discussions page for a repository
* `.` - will take you to a Visual Studio Code session in the browser for a repository, you must be signed in to use this feature

I haven't taken the time to memorize the remainder of the [shortcuts that are available](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts) but I plan on trying a few of them out. Such as `>` because I always find myself wanting to get out of VS Code and back to the GitHub page so I might as well just open it in a new tab in the first place. Another one that I just learned right now is `command shift p` that toggles between "Write" and "Preview" tabs while typing on GitHub.

**Comment by iantaylor-NOAA** [comment]
These are helpful and new to me. The one shortcut I already knew was `command shift p` which I use about 10 times every time I write an issue.
